pkg/reconciler/knativeserving/knativeserving.go

```diff
@@ -87,58 +90,71 @@ func (r *Reconciler) FinalizeKind(ctx context.Context, original *servingv1alpha1
 // converge the two.
 func (r *Reconciler) ReconcileKind(ctx context.Context, ks *servingv1alpha1.KnativeServing) pkgreconciler.Event {
 	logger := logging.FromContext(ctx)
+
 	ks.Status.InitializeConditions()
 	ks.Status.ObservedGeneration = ks.Generation
 
 	logger.Infow("Reconciling KnativeServing", "status", ks.Status)
-	stages := []func(context.Context, *mf.Manifest, *servingv1alpha1.KnativeServing) error{
-		r.ensureFinalizerRemoval,
-		r.install,
-		r.checkDeployments,
-		r.deleteObsoleteResources,
-	}
 
-	manifest, err := r.transform(ctx, ks)
+	// Get the Serving Manifest to be installed
+	servingManifest, err := r.getTargetManifest(ctx, ks)
 	if err != nil {
 		ks.Status.MarkInstallFailed(err.Error())
 		return err
 	}
 
+	// Get the previous Manifest, which has been installed.
+	oldManifest, err := r.getCurrentManifest(ctx, ks)
```

Comment:
nit: i don't love the name "old" to denote the current, working manifest. If you're ok with "target" and "current" in the function names, make the variable names match those.

Reply:
Done.